vnext/ReactUWP/Views/ScrollViewManager.h

```diff
@@ -22,19 +25,34 @@ class ScrollViewManager : public ControlViewManager
   void RemoveAllChildren(XamlView parent) override;
   void RemoveChildAt(XamlView parent, int64_t index) override;
 
+  void SnapToInterval(XamlView parent, float interval);
+  void SnapToOffsets(XamlView parent, const winrt::IVectorView<float>& offsets);
+
   void UpdateProperties(ShadowNodeBase* nodeToUpdate, folly::dynamic reactDiffMap) override;
   void DispatchCommand(XamlView viewToUpdate, int64_t commandId, const folly::dynamic& commandArgs) override;
 
 protected:
   XamlView CreateViewCore(int64_t tag) override;
 
 private:
-  void AddHandlers(winrt::Windows::UI::Xaml::Controls::ScrollViewer& scrollViewer, int64_t tag);
+  void AddHandlers(const winrt::ScrollViewer& scrollViewer, int64_t tag);
   void EmitScrollEvent(
-    winrt::Windows::UI::Xaml::Controls::ScrollViewer& scrollViewer,
+    const winrt::ScrollViewer& scrollViewer,
     int64_t tag,
     const char* eventName,
     double x, double y, double zoom);
+
+  bool m_isScrollingFromInertia{ false };
```

Comment:
we should move any local members to a shadownode just to avoid any possibility of multiple on screen instances of scrollview not picking up the same common variable here